tools/db_bench_tool.cc

```diff
@@ -190,7 +192,8 @@ DEFINE_string(
     "\tlevelstats  -- Print the number of files and bytes per level\n"
     "\tsstables    -- Print sstable info\n"
     "\theapprofile -- Dump a heap profile (if supported by this port)\n"
-    "\treplay      -- replay the trace file specified with trace_file\n");
+    "\treplay      -- replay the trace file specified with trace_file\n"
+    "\tgetmergeoperands -- Compare DB Get and GetMergeOperands performance\n");
```

Comment:
Needs clear description about what the benchmark it is. What merge operator to use. Also need to explain that binary searching the result is a part of it.